### PR TITLE
Tmo fixes

### PR DESCRIPTION
**Pull request by malw4**

### CODE REVIEW

**Comment by malw4** [comment]
@tkucma review

**Comment by malw4** [comment]
@xTrurl review

File: docs/Anjay_integrations/Anjay_ST_integration/B-L475E-IOT01A.md [hunk 1]
```diff
@@ -14,14 +14,15 @@ Integrate your B-L475E-IOT01A Discovery kit board.
 
 To get the latest binaries, navigate to [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client/releases).
 
-In this case we are going to use the demo_B-L475E-IOT01A1_merged.bin file which after being downloaded must be added to B-L475E-IOT01A board folder.
-
-To flash the board, drag the .bin file to B-L475E-IOT01A device which you can find in **devices and drives** part in **This PC**. You will see the blinking diode on your board. As soon as the diode stops blinking, the flashing has finished.
+To flash the board, open **Windows Explorer**, drag the .bin file to B-L475E-IOT01A device which you can find in **devices and drives** part in **This PC**. You will see the blinking diode on your board. As soon as the diode stops blinking, the flashing has finished.
```
Comment (mkrawczyk-avs) [inline docs/Anjay_integrations/Anjay_ST_integration/B-L475E-IOT01A.md:17]: Which .bin file would that be @malw4 ? https://github.com/AVSystem/Anjay-freertos-client/releases/download/22.09/Anjay-freertos-client-B-U585I-IOT02A-BG96.bin ?

**Review by xTrurl — COMMENTED** [review]
Reviewed. Some of the comments apply to multiple places, no use in copy-pasting them everywhere.

File: docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md [hunk 2]
```diff
@@ -45,11 +49,10 @@ Enter the command line interface on your machine, then paste and run the followi
 
 ## Flash the binaries
 To program the board, go through the whole process of **flashing Thingy:91** [described here.](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#update-nrf9160-application)
-
-After successful flashing, use the nRF Connect Programmer with the `.hex` file generated or downloaded in a previous step.
-Then reset the board and go to the next step.
+Use the nRF Connect Programmer with the downloaded `.hex` file and make [program the nRF9160 SiP application](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#program-the-nrf9160-sip-application) part.
```
[the remark this answers is not in the document]
Comment (mkrawczyk-avs) [reply docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md:52]: should the first link be to this section? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#updating-the-firmware-in-the-nrf52840-soc

File: docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md [hunk 3]
```diff
@@ -44,16 +48,14 @@ Enter the command line interface on your machine, then paste and run the followi
...
-Then reset the board and go to the next step.
+To program the board, go through the process of **flashing Thingy:91**. Use the nRF Connect Programmer with the downloaded `.hex` file and follow the [program the nRF9160 SiP application](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_thingy91_gsg.html#program-the-nrf9160-sip-application) section.
 
 
+After successful flashing, reset the board and go to the next step.
 
 ## Connect to the LwM2M Server
 
-To connect to Coiote IoT Device Management LwM2M Server, please register at https://eu.iot.avsystem.cloud/.
+To connect to Coiote IoT Device Management LwM2M Server, please register [here](https://eu.iot.avsystem.cloud/).
```
Comment (xTrurl) [inline docs/Anjay_integrations/Anjay_Nordic_integration/Thingy91.md:58]: Leave it a link, but make it clickable.
```suggestion
To connect to Coiote IoT Device Management LwM2M Server, please register [https://eu.iot.avsystem.cloud/](https://eu.iot.avsystem.cloud/).
```

File: docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md [hunk 4]
```diff
@@ -15,16 +15,17 @@ Integrate your B-L462E-CELL1 Discovery kit board along with the TYPE 1SE module
 
 To get the latest binaries, navigate to [Anjay-freertos-client](https://github.com/AVSystem/Anjay-freertos-client/releases/).
 
-In this case we are going to use the Anjay-freertos-client-B-L462E-CELL1-TYPE1SC.bin file which after being downloaded must be added to B-L462E-CELL1/TYPE1SC board folder.
-
-To flash the board, drag the .bin file to B-L462E-CELL1/TYPE1SC device which you can find in **devices and drives** part in **This PC**. You will see the blinking diode on your board. As soon as the diode stops blinking, the flashing has finished.
+To flash the board, open **Windows Explorer**, drag the .bin file to B-L462E-CELL1/TYPE1SC device which you can find in **devices and drives** part in **This PC**. You will see the blinking diode on your board. As soon as the diode stops blinking, the flashing has finished.
```
[the remark this answers is not in the document]
Comment (xTrurl) [reply docs/Anjay_integrations/Anjay_ST_integration/B-L462E-CELL1-TYPE1SC.md]: Well, **This PC** is still a Windows thing:(

**Comment by xTrurl** [comment]
I don't understand the point in renaming `Steps` to `Parts` and I still have concerns if the `making next part` really means what we want it to mean, but meh, I believe that's @mkrawczyk-avs should decide about such things.
Technically that's OK for me and I'm accepting it.

**Comment by mkrawczyk-avs** [comment]
@xTrurl It may seem a random change to rename 'Steps' to 'Parts', but what I wanted here was consistency with the text in the 'Note' about the optional "development with samples". No use explaining it here I think, but on closer inspection you should be able to find it consistent now. Also, I changed the "make next part" thing in a previous commit.

**Pull request merged**